@joselcvarela The build doesn't seem to like that 🤔 Could you check what's up with 

```
Error: src/cli/commands/instance/connect.ts(2,26): error TS7016: Could not find a declaration file for module '@directus/sdk'. '/home/runner/work/directus/directus/packages/sdk/dist/sdk.cjs.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/directus__sdk` if it exists or add a new declaration (.d.ts) file containing `declare module '@directus/sdk';`
Error: src/cli/commands/instance/disconnect.ts(2,26): error TS7016: Could not find a declaration file for module '@directus/sdk'. '/home/runner/work/directus/directus/packages/sdk/dist/sdk.cjs.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/directus__sdk` if it exists or add a new declaration (.d.ts) file containing `declare module '@directus/sdk';`
Error: src/core/extensions/instances.ts(1,26): error TS7016: Could not find a declaration file for module '@directus/sdk'. '/home/runner/work/directus/directus/packages/sdk/dist/sdk.cjs.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/directus__sdk` if it exists or add a new declaration (.d.ts) file containing `declare module '@directus/sdk';`
Error: src/core/sdk/storage.ts(1,26): error TS7016: Could not find a declaration file for module '@directus/sdk'. '/home/runner/work/directus/directus/packages/sdk/dist/sdk.cjs.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/directus__sdk` if it exists or add a new declaration (.d.ts) file containing `declare module '@directus/sdk';`
Error: src/toolbox.ts(1,57): error TS7016: Could not find a declaration file for module '@directus/sdk'. '/home/runner/work/directus/directus/packages/sdk/dist/sdk.cjs.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/directus__sdk` if it exists or add a new declaration (.d.ts) file containing `declare module '@directus/sdk';`
lerna ERR! npm run build exited 2 in '@directus/cli'
lerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.
Error: Process completed with exit code 2.
```

--- a/packages/sdk/tsconfig.json
+++ b/packages/sdk/tsconfig.json
@@ -5,6 +5,7 @@
 		"module": "es2015",
 		"moduleResolution": "node",
 		"declaration": true,
+		"declarationDir": "dist/types",
 		"declarationMap": true,
 		"strict": true,
 		"noFallthroughCasesInSwitch": true,

